pkg/operator/operator.go

```diff
@@ -325,6 +327,7 @@ func (optr *Operator) sync(key string) error {
 		templatectrl.SetupEtcdEnvKey:         imgs.SetupEtcdEnv,
 		templatectrl.InfraImageKey:           imgs.InfraImage,
 		templatectrl.KubeClientAgentImageKey: imgs.KubeClientAgent,
+		templatectrl.EtcdQuorumGuardImageKey: imgs.EtcdQuorumGuardImage,
```

Comment:
You have to add your new sync function below here with the others (not sure why you didn't) (this comment is not related to this line of course) specifically, you need syncEtcdQuorumGuard in the list with the others

Reply:
Yep.